uktrade/data-hub-frontend
# Feature/DET-242 Add more sort-by options to Aventri attendees page

## Description of change

At the moment, we can sort the Aventri attendees page by First name (A-Z or Z-A). This PR will enable us to also sort by Last name and Company name (both A-Z or Z-A)

## Test instructions

- Ensure you have user-event-activities feature flag on. You can do this is dev django admin.
- Get this branch up locally
- Visit this Aventri event page : http://localhost:3000/events/aventri/3333/attendees?featureTesting=user-event-activities
- You should be able to see the ability to sort by First name, Last name, Company name

_What should I see?_

## Screenshots
### Before

![image](https://user-images.githubusercontent.com/83657534/184662239-8cc906b0-79bb-4f92-933f-2f69006968bc.png)



### After
![image](https://user-images.githubusercontent.com/83657534/184650929-5857eb7e-67a2-468d-8d0d-d4ba176194a0.png)


## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
